docs/designs/crd.md

```diff
@@ -401,3 +406,48 @@ Every NodePool is defining different Opensearch Nodes StatefulSet
         <td> - </td>
       </tr><tr>
 </table>
+
+
+<h3 id="GeneralConfig">
+  Monitoring
+</h3>
+
+Monitoring defines Opensearch monitoring stuck
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+            <th>default</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>enable</b></td>
+        <td>bool</td>
+        <td>Define if to ernable monitoring for that cluster</td>
+        <td>true</td>
+        <td>-</td>
+      </tr><tr>
+        <td><b>monitoringUser</b></td>
+        <td>[]string</td>
+        <td>Define from which user the monitor will run (the user should be created earlier).</td>
+        <td>false</td>
+        <td>admin</td>
+      </tr><tr>
+        <td><b>interval</b></td>
+        <td>string</td>
+        <td>Define interval for scarping</td>
+        <td>false</td>
+        <td>30s</td>
+      </tr><tr>
+      </tr><tr>
+        <td><b>offlinePlugin</b></td>
+        <td>string</td>
+        <td>Define offline link to Aiven Plugin</td>
+        <td>false</td>
+        <td>https://github.com/aiven/prometheus-exporter-plugin-for-opensearch</td>
```

Comment:
The example URL should be a complete path with versions and all as the operator expects it. Also missing is info that the plugin must match the opensearch version.